Samples/blazor-add-in/word-blazor-add-in/README.md

```diff
@@ -61,7 +61,7 @@ protected override async Task OnAfterRenderAsync(bool firstRender)
 {
   if (firstRender)
   {
-    JSModule = await JSRuntime.InvokeAsync<IJSObjectReference>("import", "./Pages/Index.razor.js");
+    JSModule = await JSRuntime.InvokeAsync<IJSObjectReference>("import", "./Pages/HelloWorld.razor.js");
```

Comment:
There isn't a page named HelloWorld that I can see.

Reply:
Yes, there is (note that this changes the WORD sample README.md) 😉

The error fixed here is related to the text -around- the example that is explaining about HelloWorld pages but the example had Index instead:

![image](https://user-images.githubusercontent.com/5305682/188325930-92802e9b-b4de-474f-a9ea-8e15cfcf7e07.png)

so even though it is unrelated to the Outlook project I sneaked this in to fix the example 😁